d2l-image.js

```diff
@@ -16,66 +32,59 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-image">
 				border-radius: var(--d2l-image-border-radius);
 				object-fit: var(--d2l-image-object-fit);
 			}
-		</style>
+		`;
+	}
 
-		<iron-ajax id="imageRequest" url="[[imageUrl]]" headers="[[_headers]]" on-iron-ajax-response="_onImageResponse" on-iron-ajax-error="_onImageResponse" handle-as="blob">
-		</iron-ajax>
+	constructor() {
+		super();
+		window.d2lfetch.use({ name: 'auth', fn: auth });
```

Comment:
I'm now a little worried about this... since it's a global this is essentially adding the auth middleware to any other fetch usage on the same page. I think `addTemp` might safer?
https://github.com/Brightspace/d2l-fetch#addtemp